yarn-project/prover-client/src/orchestrator/orchestrator.ts

```diff
@@ -193,6 +209,9 @@ export class ProvingOrchestrator {
    * The interface to add a simulated transaction to the scheduler
    * @param tx - The transaction to be proven
    */
+  @trackSpan('ProvingOrchestrator.addNewTx', tx => ({
+    [Attributes.TX_HASH]: tx.hash.toString(),
```

Comment:
Question: what's the purpose of the span attributes? Is it meant to be used as an identifier, or just as a dimension for categorization? Asking because some platforms dislike having attributes with too high a cardinality, not sure about Jaeger here.

Reply:
That's a good point, but I think in this case the tx hash is our request id from distributed system. I'll remove the other ones (block size, pending txs, etc, but keep circuit name)